components/PointsCount/__tests__/index.js

```diff
@@ -0,0 +1,27 @@
+import { mount } from 'enzyme'
+import React from 'react'
+import PointsCount from '../'
+
+function pointsCount(count) {
```

Comment:
Wonder if there's a better function name, which more clearly labels what it does?